src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/ExpressRouteCircuitTests.cs

```diff
@@ -40,7 +40,7 @@ public void TestExpressRouteCircuitPrivatePublicPeeringCRUD()
             NetworkResourcesController.NewInstance.RunPsTest("Test-ExpressRouteCircuitPrivatePublicPeeringCRUD");
         }
 
-        [Fact]
+        [Fact(Skip = "Temporary disabled: no feature on subscription to re-record")]
```

Comment:
Waiting on these test to be rerecorded by the Express Route team: @akshaysngupta if you have access to subscription 947d47b4-7883-4bb9-9d85-c5e8e2f572ce can you rerecord these three tests with this subscription?  Thanks.